tools/maint/rebaseline_tests.py

```diff
@@ -37,6 +37,15 @@ def run(cmd, **args):
 all_deltas = []
 
 
+def read_size_from_json(content):
+  json_data = json.loads(content)
+  if 'total' in json_data:
+    return json_data['total']
+  # If `total` is not in the json dict then just use the firt key
+  first_key = list(json_data.keys())[0]
```

Comment:
is converting a dict to a list like this deterministic?

Reply:
Hmm, iirc dicts are insertion-order since Python 3.7, but we support 3.6 still, so maybe this should be sorted?

Reply:
We depend on 3.8 at least

Reply:
@sbc100 oh, then is this out of date, where it says 3.6?

https://emscripten.org/docs/building_from_source/toolchain_what_is_needed.html#emscripten-tools-and-dependencies

Reply:
(but lgtm for this PR)

Reply:
Ah yes, that is out of date.   We updated to 3.8 in #23417, and then updated some docs in #24422, but missed that one.